planetiler-custommap/planetilerspec.schema.json

```diff
@@ -83,6 +83,10 @@
           "description": "Max zoom level that the output feature appears in",
           "type": "integer"
         },
+        "min_pixel_size": {
+          "description": "Minimum length of line features or square root of the minimum area of polygon features to emit below the maximum zoom-level of the map",
+          "type": "double"
+        },
```

Comment:
This schema is actually for the tests for a schema, it's good to include here but can you make the description consistent with the other fields and also add to `planetiler.schema.json` (the type should be `"$ref": "#/$defs/expression"` like `min_zoom` since you can specify it with an expression as well as a constant)